action/protocol/execution/evm/evm.go

```diff
@@ -475,16 +492,20 @@ func evmErrToErrStatusCode(evmErr error, g genesis.Blockchain, height uint64) (e
 }
 
 // intrinsicGas returns the intrinsic gas of an execution
-func intrinsicGas(data []byte) (uint64, error) {
+func intrinsicGas(size uint64, list types.AccessList) (uint64, error) {
 	if action.ExecutionDataGas == 0 {
 		panic("payload gas price cannot be zero")
 	}
-	dataSize := uint64(len(data))
-	if (math.MaxInt64-action.ExecutionBaseIntrinsicGas)/action.ExecutionDataGas < dataSize {
+
+	var accessListGas uint64
+	if len(list) > 0 {
+		accessListGas = uint64(len(list)) * action.TxAccessListAddressGas
```

Comment:
`+=`

Reply:
here is the first assignment, so can use `=`, the next line use `+=`